list.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
-var github = require('github-repositories')
 var Promise = require('promise')
+var github = Promise.denodeify(require('github-repositories'))
```

Comment:
I like where you're going with this, but [`Promise.denodeify`](https://www.npmjs.com/package/promise#promise-denodeify-fn) isn't quite considered standard ES6. Could we use something that works with straight up Promises?